src/init/ossec-local.sh

```diff
@@ -140,31 +146,20 @@ disable()
 {
     if [ "X$2" = "X" ]; then
         echo ""
-        echo "Disable options: database, client-syslog, agentless, debug, integrator"
-        echo "Usage: $0 disable [database|client-syslog|agentless,debug|integrator]"
+        echo "Disable options: debug"
+        echo "Usage: $0 disable debug"
         exit 1;
     fi
     daemon=''
-    if [ "X$2" = "Xdatabase" ]; then
-        echo "DB_DAEMON=\"\"" >> ${PLIST};
-        daemon='ossec-dbd'
-    elif [ "X$2" = "Xclient-syslog" ]; then
-        echo "CSYSLOG_DAEMON=\"\"" >> ${PLIST};
-        daemon='ossec-csyslogd'
-    elif [ "X$2" = "Xagentless" ]; then
-        echo "AGENTLESS_DAEMON=\"\"" >> ${PLIST};
-        daemon='ossec-agentlessd'
-    elif [ "X$2" = "Xintegrator" ]; then
-        echo "INTEGRATOR_DAEMON=\"\"" >> ${PLIST};
-        daemon='ossec-integratord'
-    elif [ "X$2" = "Xdebug" ]; then
+
+    if [ "X$2" = "Xdebug" ]; then
         echo "DEBUG_CLI=\"\"" >> ${PLIST};
-    else
+    elif [ "X$2" != "Xdatabase" ] && [ "X$2" != "Xclient-syslog" ] && [ "X$2" != "Xagentless" ] && [ "X$2" != "Xintegrator" ]; then
```

Comment:
For the disable option, set descriptive messages in the same way as when enabling them.